services/ad-server/src/public/ad-frame.html

```diff
@@ -46,7 +54,15 @@
           frameEl.src = auctionResult
         }
 
-        frameEl.className = 'ad'
+        if (isVideo && auctionResult) {
```

Comment:
Nit: Should we reverse order of elements, i.e. make it "if (auctionResult && isVideo)"? 
This would match the order as previous if-statements.

Reply:
Yea, that makes it cleaner.  Switched the order.